pytket/qir/conversion/conversion.py

```diff
@@ -424,11 +424,11 @@ def _get_optype_and_params(self, op: Op) -> tuple[OpType, Sequence[float]]:
         params: list = []
         if optype == OpType.ExplicitPredicate:
             if op.get_name() == "AND":
-                optype = BitWiseOp.AND
+                optype = BitWiseOp.AND  # type: ignore
```

Comment:
Mypy is not able to see that BitWiseOp.XOR is an optype.

Reply:
But it is actually a different enum, is this variable maybe mistyped?